pkg/github/github.go

```diff
@@ -0,0 +1,99 @@
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"time"
+
+	"github.com/astronomerio/astro-cli/pkg/httputil"
+	"github.com/pkg/errors"
+)
+
+var (
+	http = httputil.NewHTTPClient()
```

Comment:
@cwurtz How do you feel about having these at the API level instead of moving them into the cmd level like we have it with houston? This allows us to treat the `github` package as a complete wrapper to the API, not having to worry about client instantiation